app/mailers/user_mailer.rb

```diff
@@ -422,6 +400,17 @@ def open_project_headers(hash)
 # Unfortunately, this results in changes on the interceptor classes during development mode
 # not being reflected until a server restart.
 
+class PlainTextInterceptor
+  def self.delivering_email(message)
+    if Setting.plain_text_mail?
+      # HACK: this is a workaround to ensure @html_part ivar is set correctly
+      # on the Mail::Message.
+      message.html_part = message.html_part
+      message.html_part = nil
```

Comment:
@NobodysNightmare this was intended. For the full story, you'll need to dive into the mail source-code, but suffice to say [`Mail::Message#html_part=`](http://www.rubydoc.info/github/mikel/mail/Mail/Message:html_part=) does more than your typical accessor method. For some reason, Rails doesn't set up internal state correctly (i.e. set an `@html_part` ivar). This works around this.